Lib/test/test_tcl.py

```diff
@@ -736,8 +736,11 @@ def check(value):
         check('{\n')
         check('}\n')
 
+    @support.cpython_only
     def test_new_tcl_obj(self):
         self.assertRaises(TypeError, _tkinter.Tcl_Obj)
```

Comment:
What do you think of adding `support.assert_disallow_instanciation(type, *args)` function which would check for TypeError but also check the error message? TypeError is quite generic and it might hide a bug.

Reply:
I think it's a good idea!

Reply:
Let's add it in a separate PR. I guess we can reuse the bpo number.